### PR TITLE
Fix TV and QC Surveys rendering and add Sentry Logs

### PR DESCRIPTION
**Pull request by kepstein23**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Game } from "./Game";
 import IntroCheck from "./intro-exit/IntroCheck";
 import { EnterNickname } from "./intro-exit/EnterNickname";
 import VideoCheck from "./intro-exit/VideoCheck";
-import { usePlayer, useGame, useStage, useRound } from "@empirica/player";
+import { usePlayer, useGame, useStage, useRound, useGlobal } from "@empirica/player";
```
Comment (JamesPHoughton) [inline client/src/App.jsx:8]: If we're not using this anymore, lets pull it out

File: client/src/components/SurveyWrapper.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { useCallback } from "react";
+import React, { useCallback, useRef, useEffect } from "react";
```
Comment (JamesPHoughton) [inline client/src/components/SurveyWrapper.jsx:1]: are useRef and useEffect needed here?

File: server/src/callbacks.mjs [hunk 3]
```diff
@@ -37,6 +37,8 @@ function validateURL(url) {
 
 Empirica.onGameStart(function ({ game }) {
   console.log("game start");
+  console.log("game." + game.batch.get("TVSurveys")[game.treatment.TVSurvey] != null)
```
Comment (JamesPHoughton) [inline server/src/callbacks.mjs:40]: this is a bit of an ambiguous message, isn't it? "game.true"?

**Review by JamesPHoughton — CHANGES_REQUESTED** [review]

File: server/src/callbacks.mjs (hunk 3 above)
Comment (kepstein23) [reply server/src/callbacks.mjs:40]: oops, I meant to delete that it was for debugging

**Pull request merged**